prow/pubsub/reporter/reporter.go

```diff
@@ -106,9 +120,11 @@ func (c *Client) Report(pj *kube.ProwJob) error {
 }
 
 func (c *Client) generateMessageFromPJ(pj *kube.ProwJob) *ReportMessage {
-	projectName := pj.Labels[PubSubProjectLabel]
-	topicName := pj.Labels[PubSubTopicLabel]
-	runID := pj.GetLabels()[PubSubRunIDLabel]
+	pubSubMap := findLabels(pj, PubSubProjectLabel, PubSubTopicLabel, PubSubRunIDLabel)
+
+	projectName := pubSubMap[PubSubProjectLabel]
+	topicName := pubSubMap[PubSubTopicLabel]
+	runID := pubSubMap[PubSubRunIDLabel]
 
 	psReport := &ReportMessage{
 		Project: projectName,
```

Comment:
Seems like we could just update this and remove the projectName var?

Reply:
done